weather.ozweather/resources/lib/common.py

```diff
@@ -33,7 +33,7 @@
 """
 Determine if we are unit testing outside of Kodi, or actually running within Kodi
 Because we're using Kodi stubs https://romanvm.github.io/Kodistubs/ we can't rely on 'import xbmc' failing
-So we use this hack - Kodi will return a user agent string, but Kodistrubs just returns and empty string.
+So we use this hack - Kodi will return a user agent string, but Kodistubs just returns an empty string.
```

Comment:
Just so you know: you could use https://pypi.org/project/sakee/ for Kodi stubbing. It actually tries to mimic more the Kodi functionality.